juriscraper/pacer/utils.py

```diff
@@ -161,6 +161,11 @@ def is_pdf(response):
         return True
     return False
 
+def is_text(response):
+    """Determines whether the item downloaded is a text file or something else."""
+    if '.txt' in response.headers.get("content-type"):
```

Comment:
Should this more closely mirror the `is_pdf` function's if statement?

Reply:
The issue is that when there is a text file, the `content-type` is something like `'application/binary; name=CaseFiledRPT.txt'`. In other cases where a text file is returned, like in the claims registry, it may have a different file name than `CaseFileRPT.txt`. Hence why I was checking for ".txt" instead of doing a direct comparison like in the `is_pdf` function.

Reply:
Got it. In that case, I'd just want to guard against that header being missing. For example, if headers.get("foo") returns `None`, you'll get an exception. Better to either do:

`headers.get("foo", "")`

Or:

`if headers.get("foo") and "xyz" in headers.get("foo"):`

Either way is probably fine, I'd say, so long as we don't rely on that header always being there.

Reply:
Good point. Hadn't thought of that. Added `headers.get("foo", "")` in newest commit.